schematic/models/validate_manifest.py

```diff
@@ -216,9 +218,12 @@ def validate_manifest_rules(
 
                     t_indiv_rule = perf_counter()
                     # Validate for each individual validation rule.
-                    validation_method = getattr(
-                        ValidateAttribute, validation_types[validation_type]["type"]
-                    )
+                    try:
+                        validation_method = getattr(
+                            ValidateAttribute, validation_types[validation_type]["type"]
+                        )
+                    except:
+                        breakpoint()
```

Comment:
I'm assuming this breakpoint got left in by accident?

Reply:
Yes, I forgot that was there! thank you!